src/main/java/org/killbill/billing/plugin/adyen/client/payment/service/AdyenPaymentServiceProviderHostedPaymentPagePort.java

```diff
@@ -100,9 +100,14 @@ public Map getDirectory(final String merchantAccount,
         return directoryClient.getDirectory(params);
     }
 
-    // Used to verify completion
-    @SuppressWarnings("deprecation")
+
     public HppCompletedResult parseAndVerifyRequestIntegrity(final Map<String, String> requestParameterMap) {
+        return parseAndVerifyRequestIntegrity(requestParameterMap, adyenConfigProperties, logger);
+    }
+
+    public static HppCompletedResult parseAndVerifyRequestIntegrity(final Map<String, String> requestParameterMap,
+                                                                    final AdyenConfigProperties adyenConfigProperties,
```

Comment:
Why do you need to pass these `properties`? They should be the same as the one from `AdyenPaymentServiceProviderHostedPaymentPagePort` (same tenant).

Reply:
See b08b884. Removed the change that added this static method.